src/gameTest/java/mekanism/common/tests/energy/EnergyConversionTest.java

```diff
@@ -0,0 +1,194 @@
+package mekanism.common.tests.energy;
+
+import mekanism.api.Action;
+import mekanism.api.energy.IStrictEnergyHandler;
+import mekanism.common.capabilities.energy.BasicEnergyContainer;
+import mekanism.common.config.MekanismConfig;
+import mekanism.common.integration.energy.forgeenergy.ForgeStrictEnergyHandler;
+import mekanism.common.tests.helpers.EnergyTestHelper;
+import net.minecraft.gametest.framework.GameTest;
+import net.neoforged.neoforge.energy.EnergyStorage;
+import net.neoforged.neoforge.energy.IEnergyStorage;
+import net.neoforged.testframework.annotation.ForEachTest;
+import net.neoforged.testframework.annotation.TestHolder;
+import net.neoforged.testframework.gametest.EmptyTemplate;
+
+@ForEachTest(groups = "energy.conversion")
+public class EnergyConversionTest {
+
+    @GameTest
+    @EmptyTemplate
+    @TestHolder(description = "Tests that energy conversion works properly with various decimal values.")
+    public static void testForgeEnergy(final EnergyTestHelper helper) {
+        double defaultConversion = 2.5;
+        double inverseConversion = 1 / defaultConversion;
+        int capacity = 1_000;
+        int convertedCapacity = (int) (capacity / defaultConversion);
```

Comment:
I think for a test, this should be a fixed value, and the calculation in a comment. Personally I'd also put them in screaming snake case for readability